modules/ROOT/pages/advanced/annotations.adoc

```diff
@@ -30,7 +30,7 @@ setup: function(ed) {
 }
 ----
 
-See link:{baseurl}/configure/[Configure {productname}] for more information on how to configure {productname} core.
+See xref:/configure.adoc[Configure {productname}] for more information on how to configure {productname} core.
```

Comment:
This xref doesn't resolve... I believe it should be this instead:
```suggestion
See xref:configure/index.adoc[Configure {productname}] for more information on how to configure {productname} core.
```

Reply:
@lnewson I think this is one of that cases when the page does not exist anymore, it could make sense to redirect to: `configure/integration-and-setup.html`  (first topic of: `Configuration reference`) ? `configure/index.adoc` is an empty page and also do not highlight any voice of the left menu

Reply:
It's not that it doesn't exist, it's that it still needs determining what to do with it since the old index page was generated dynamically. There's a good chance we'll need to manually recreate those index pages, but that's something that still needs a little more discussion. For now, we should keep the links pointing there and then we'll either add the content to the index pages or delete them and work with the docs team to determine where they should point.

Reply:
Ok, thanks for the explanation, I'll change it

Reply:
done